generate/generate.go

```diff
@@ -334,6 +334,17 @@ func (g *Generator) ClearProcessEnv() {
 // AddProcessEnv adds env into g.spec.Process.Env.
 func (g *Generator) AddProcessEnv(env string) {
 	g.initSpec()
+	// I'm not sure what an environment variable without an = in it means, but
+	// better safe than sorry.
+	if strings.Contains(env, "=") {
```

Comment:
I'd rather error if the string does not contain an equal sign.  From [POSIX][1]:

> These strings have the form *name*=*value*…

[1]: http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap08.html